docs/process-services-cloud/components/form-cloud.component.md

```diff
@@ -160,6 +163,49 @@ For an existing Task both the form and its values will be fetched and displayed.
 
 In this case, only the form definition will be fetched.
 
+#### Enabling fullscreen display for the form
+
+Provide a `displayModeConfiguration` array object containing the fullscreen configuration. You can use the configuration provided in the [`CloudFormRenderingService`](../../../lib/process-services-cloud/src/lib/form/components/cloud-form-rendering.service.ts) as a static member `CloudFormRenderingService.IMPLEMENTED_DISPLAY_MODE_CONFIGURATIONS`, or configure your own if you want to customise the options for the fullscreen display mode.
+
+**MyView.component.html**
+
+```html
+<button (click)="displayFormInFullscreenMode()">
```

Comment:
this example can be simplified, you don't need `@ViewChild` and extra methods, as you already provide reference to the form as `#adfCloudForm`, so the code can be simply:

```html
<button (click)="adfCloudForm.switchToDisplayMode('fullScreen')">Full screen</button>
```

Reply:
Thanks! I will apply this!